headers/allsky.header

```diff
@@ -0,0 +1,59 @@
+ORIGIN  = 'LSST Generic Camera Service'/ FITS file originator
```

Comment:
I recently have been struggling with how conda manage "data_files". I think it may be better to have them a "package_data". The main difference is that a "package_data" resides with the modules and they are marked as part of the module by simply adding an `__init__.py` file to the directory.

Consider moving `header/` to `python/lsst/ts/GenericCamera/header/` and adding an `__init__.py` file in there. Then, update the `setup.py` with:

```
    include_package_data=True,
    package_data={
        "": ["*.header"],
    },
```

Reply:
I was struggling with where to put the header files so thanks for the great idea. Done.